index.bs

```diff
@@ -1117,6 +1114,11 @@ A {{SendStream}} has the following internal slots.
  <tbody>
 </table>
 
+Note: These internal slots need to be attached to {{SendStream}} (which is a kind of
+{{WritableStream}}s) conceptually, not phisically. For example, implementers MAY have a weak hash
```

Comment:
Maybe the "s" in `{{WritableStream}}s` isn't needed.

Spelling: "physically".

I don't think you should use `MAY` within a note section.

In general, I would prefer to recommend an implementation strategy where the slots are owned by an object which is owned by the algorithms.

Something like
```
For example, an implementation could place them on an object which is
referenced from writeAlgorithm, closeAlgorithm and abortAlgorithm.
```

Reply:
Fixed nits: thank you!

> In general, I would prefer to recommend an implementation strategy where the slots are owned by an object which is owned by the algorithms.

I can do that: Just to confirm, are you fine with the recommendation with https://w3c.github.io/webtransport/#send-stream-STOP_SENDING?

Reply:
I'm confused. That algorithm references a "remoteCanceled" promise which isn't defined anywhere.

Reply:
Ah sorry I forgot to remove the step. I'll do that separately.

What I'm talking about here is the reference to the `[[Transport]]` internal slot at the first step.

> 1. Let _transport_ be _stream_’s [[Transport]].

Reply:
I think that's okay. It's not ambiguous.

Reply:
Fixed.